Kudu.Contracts/SiteExtensions/SiteExtensionInfo.cs

```diff
@@ -40,6 +40,7 @@ public SiteExtensionInfo(SiteExtensionInfo info)
             LocalIsLatestVersion = info.LocalIsLatestVersion;
             LocalPath = info.LocalPath;
             InstalledDateTime = info.InstalledDateTime;
+            InstallationArgs = info.InstallationArgs;
```

Comment:
What will be the behavior when I change the value in ARM? Will it kick off redeployment of and extension?